test/cmd/newapp.sh

```diff
@@ -12,6 +12,7 @@ os::log::install_errexit
 # This test validates the new-app command
 
 os::cmd::expect_success_and_text 'oc new-app library/php mysql -o yaml' '3306'
+os::cmd::expect_success_and_text 'oc new-app library/php mysql --dry-run' "Image \"mysql\" runs as the 'root' user which may not be permitted by your cluster administrator"
```

Comment:
one is checking for text to be present, which needs to be very specific.

the other is checking that the text not contain a string, in which case less specific is better (if the image name changed, the test would start passing, which is undesirable)